filtered-resource-service/src/component-tests/java/uk/gov/gchq/palisade/component/filteredresource/message/AuditErrorMessageTest.java

```diff
@@ -33,26 +35,30 @@
 import static org.assertj.core.api.Assertions.assertThat;
 import static org.junit.jupiter.api.Assertions.assertAll;
 
-@RunWith(SpringRunner.class)
 @JsonTest
-public class AuditErrorMessageTest {
+@ContextConfiguration(classes = {FilteredResourceApplication.class})
+class AuditErrorMessageTest {
 
     @Autowired
     private JacksonTester<AuditErrorMessage> jsonTester;
 
+    @Test
+    void contextLoads() {
```

Comment:
naming convention for tests should be test*.java
```suggestion
    void testContextLoads() {
```